cfme/tests/cloud_infra_common/test_quota_manual.py

```diff
@@ -110,8 +105,6 @@ def test_instance_quota_reconfigure_with_flavors():
     pass
 
 
-@pytest.mark.manual
-@test_requirements.service
```

Comment:
Deleted `test_requirements.service` by mistake?

Reply:
No. This test case comes under ```quota```. hence intentionally deleted.